dashboard/modules/usage_stats/usage_stats_head.py

```diff
@@ -19,6 +21,9 @@ def __init__(self, dashboard_head):
             ray.experimental.internal_kv.internal_kv_get_gcs_client(),
             num_retries=20,
         )
+        self.cluster_config = ray_usage_lib.get_cluster_config(
```

Comment:
Is it possible to make it async? Any blocking API inside the dashboard can cause stability issues (or unexpected delay in other important operations)

Reply:
Does async help here?
(Is there any logic within this method that has the potential to yield to another coroutine?)

There's related logic here https://github.com/ray-project/ray/blob/678d23fe42196583a4996374d642feee78778c07/dashboard/modules/reporter/reporter_head.py#L118
But I don't really see how the async helps there either, since the internal kv call is synchronous.

Reply:
I think you can either support async internal kv or post it to a thread pool

Reply:
Not to revive the old discussion, but if we just put it in a separate thread initially this wouldn't have been an issue, no?

Reply:
Put report in a separate thread.